src/Components/AllSightings.js

```diff
@@ -13,27 +13,30 @@ const AllSightings = () => {
 
   return (
     <>
-      <div className='search-container'>
-        <div>
-          <button
-            className='cryptid-button'
-            onClick={toggleDisplay}
-          >
-            Cryptids
-          </button>
-          {display &&
-            <div className='dropdown-container'>
-              <p>things go here</p>
-            </div>
-          }
+      <div className='sightings-header'>
```

Comment:
Love these changes! It's looking so good! Great idea from Jasmyn to add the page title!